src/Routes.js

```diff
@@ -35,6 +36,7 @@ function checkPaths(technology, app) {
 
 export const Routes = ({ childProps: { history } }) => {
     const pathName = window.location.pathname.split('/');
+    const { status } = queryString.parse(location.search);
```

Comment:
did you consider using brosers `URLSearchParams` instead of the library?

Reply:
Good point! This library looks slightly easier to use, but better to use browser build in features than custom libraries.